src/libs/PolicyUtils.ts

```diff
@@ -874,11 +929,19 @@ export {
     findCurrentXeroOrganization,
     getCurrentXeroOrganizationName,
     getXeroBankAccountsWithDefaultSelect,
+    findSelectedVendorWithDefaultSelect,
+    findSelectedPayableAccountWithDefaultSelect,
+    findSelectedReceivableAccountWithDefaultSelect,
```

Comment:
I believe this one is only used within this file here, so we don't need to export it.